provisioning/device/src/Transports/Amqp/AmqpClientConnection.cs

```diff
@@ -17,6 +16,13 @@ namespace Microsoft.Azure.Devices.Provisioning.Client
 {
     internal sealed class AmqpClientConnection : IDisposable
     {
+        private const string Amqpwsb10 = "AMQPWSB10";
+        private const string UriSuffix = "/$iothub/websocket";
+        private const string Scheme = "wss";
+        private const string Version = "13";
+        private const int WebSocketPort = 443;
+        private const int TcpPort = 5671;
```

Comment:
Deleted the constants file that these lived in before since this was the only class referencing them